src/Moryx.Runtime.Endpoints/Modules/Endpoint/ModulesController.cs

```diff
@@ -254,7 +256,7 @@ private ICustomSerialization CreateSerialization(IModule module)
         {
             var host = (IContainerHost)module;
             // TODO: This is dangerous
```

Comment:
Why is this dangerous? 🤔

Reply:
Because we just assume that the config manager is an `IEmptyPropertyProvider`.